### PR TITLE
Top Level Unstructured Diff

### PR DESCRIPTION
**Pull request by gme5078**
Diff for top level Unstructured Profile

### CODE REVIEW

**Comment by gme5078** [comment]
Work in progress because I am going to partially abstract it. And I need type instance checking

File: dataprofiler/tests/profilers/test_profile_builder.py [hunk 1]
```diff
@@ -1670,6 +1670,60 @@ def test_merge_profiles(self, *mocks):
...
+
+        data1 = pd.Series(['this', 'is my', '\n\r', 'test'])
+        data2 = pd.Series(['here\n', '\t    ', ' ', ' is', '\n\r', 'more data'])
+        profiler1 = UnstructuredProfiler(data1)
+        profiler2 = UnstructuredProfiler(data2)
+
+        expected_diff = {
+            'global_stats': {
+                'samples_used': -2, 
+                'empty_line_count': -5,
```
Comment (AnhTruong) [inline dataprofiler/tests/profilers/test_profile_builder.py:1704]: should be -3?

Comment (gme5078) [reply dataprofiler/tests/profilers/test_profile_builder.py]: Good catch, fixed

File: dataprofiler/tests/profilers/test_profile_builder.py [hunk 2]
```diff
@@ -1907,6 +1907,60 @@ def test_merge_profiles(self, *mocks):
...
+
+        data1 = pd.Series(['this', 'is my', '\n\r', 'test'])
+        data2 = pd.Series(['here\n', '\t    ', ' ', ' is', '\n\r', 'more data'])
+        profiler1 = UnstructuredProfiler(data1)
+        profiler2 = UnstructuredProfiler(data2)
+
+        expected_diff = {
+            'global_stats': {
+                'samples_used': -2, 
+                'empty_line_count': -2,
```
Comment (lettergram) [inline dataprofiler/tests/profilers/test_profile_builder.py:1941]: Why did this error, shouldn't this be tested seperately?

File: dataprofiler/tests/profilers/test_profile_builder.py [hunk 3]
```diff
@@ -1907,6 +1907,60 @@ def test_merge_profiles(self, *mocks):
...
+        profiler1 = UnstructuredProfiler(data1)
+        profiler2 = UnstructuredProfiler(data2)
+
+        expected_diff = {
+            'global_stats': {
+                'samples_used': -2, 
+                'empty_line_count': -2, 
+                'file_type': 'unchanged', 
+                'encoding': 'unchanged', 
+                'memory_size': -9.5367431640625e-06
```
Comment (JGSweets) [inline dataprofiler/tests/profilers/test_profile_builder.py:1944]: `-10/1024**2`

**Review by JGSweets — APPROVED** [review]

**Review by AnhTruong — APPROVED** [review]

**Pull request merged**